cross_list_courses/static/cross_list_courses/js/app.js

```diff
@@ -0,0 +1,39 @@
+(function(){
+    var app = angular.module('CrossListCourses',
+                             ['ngSanitize', 'ng.django.urls', 'ngRoute',
+                              'ui.bootstrap', 'datatables']);
+    
+    app.config(['$httpProvider', '$routeProvider',
+                function($httpProvider, $routeProvider){
+        $httpProvider.defaults.headers.common['X-Requested-With'] = 'XMLHttpRequest';
+        $httpProvider.defaults.xsrfCookieName = 'csrftoken';
+        $httpProvider.defaults.xsrfHeaderName = 'X-CSRFToken';
+        $httpProvider.interceptors.push(function(){
+            return {
+                'request': function(config){
```

Comment:
Thinking out loud, I wonder, given we use this everywhere now, if we can get it into a common library and not have to define it in every new app. Not for this PR, but worth a thought for the future, just marking it down somewhere.